### DISCUSSION TITLE
Can't connect to external mysql instance on podman 5.2.2

### DISCUSSION THREAD

**Discussion by beeritis** [containers/podman#24641] [General]
Hi,

We've had  a recent upgrade to podman 5.2.2 breaks a container running apache from being able to connect to a mysql instance running on the same host as the container (I know mysql should be in a container as well but this is how it  has been set up)

We get "Connection refused" using podman `5.2.2` but if we downgrade to `4.9.4` it works perfectly fine. 

Been through the release notes but couldn't find anything, unless I missed it. 

If anyone has any ideas that would be great . 

Thamks,

**Comment by beeritis** [comment]
Figured it out , because the default network type is now "pasta" , for whatever reason it blocks then connection. 
If I run the container with `--net=slirp4netns` it works as it did before. 

Any suggestions on how to get this working with "pasta" as it's described to have "improved performance" . 
Thanks!

**Reply by Luap99** [reply]
https://github.com/containers/podman/issues/24613#issuecomment-2486672230

**Reply by beeritis** [reply]
Thanks for that 👍  . Looks like I'll need to wait for podman 5.3 to become available (we use RHEL9) so will wait for it to appear as a package update in the repos.  `slirp4netns` works as a workaround in the meantime.

**Reply by Luap99** [reply]
Note you still will have to switch to `host.containers.internal` as the same ip problem will always exists with the default pasta settings.
